File: examples-sdk/swift/206-llm-decision-routing/main.swift
// 206-llm-decision-routing — Swift SDK equivalent
// Compile: vil compile --from swift --input 206-llm-decision-routing/main.swift --release

let server = VilServer(name: "insurance-underwriting-ai", port: 3116)
let underwriter = ServiceProcess(name: "underwriter")
server.service(underwriter)
server.compile()

File: examples-sdk/kotlin/206-llm-decision-routing/main.kt
// 206-llm-decision-routing — Kotlin SDK equivalent
// Compile: vil compile --from kotlin --input 206-llm-decision-routing/main.kt --release

fun main() {
    val server = VilServer("insurance-underwriting-ai", 3116)
    val underwriter = ServiceProcess("underwriter")
    server.service(underwriter)
    server.compile()
}
